src/examples/renaming_and_moving_objects.cpp

```diff
@@ -0,0 +1,70 @@
+#include <H5File.hpp>
+#include <H5Group.hpp>
+#include <H5DataSet.hpp>
+#include <H5DataSpace.hpp>
+#include <H5Attribute.hpp>
```

Comment:
This causes the error in the CI:
```suggestion
#include <highfive/H5File.hpp>
#include <highfive/H5Group.hpp>
#include <highfive/H5DataSet.hpp>
#include <highfive/H5DataSpace.hpp>
#include <highfive/H5Attribute.hpp>
```